lib/src/injector_base.dart

```diff
@@ -6,49 +6,45 @@ import 'package:injector/src/factory/provider_factory.dart';
 import 'package:injector/src/factory/singelton_factory.dart';
 
 class Injector {
-  /**
-   * The static / single instance of the injector
-   */
+  /// The static/single instance of the [Injector].
   static final Injector appInstance = Injector();
 
-  /**
-   * The constructor to create a new instance
-   */
-  Injector();
-
-  /**
-   * The Map that contains:
-   *
-   *  [SingletonFactory] - The single instances
-   *
-   * [ProviderFactory] - The provider factory that contains the type information.
-   *                     If creates a new instance every time
-   */
+  /// Stores [SingletonFactory] and [ProviderFactory] instances that have been
+  /// registered by [registerSingleton] and [registerDependency] respectively.
```

Comment:
Do we expose `Factory` and a way to register custom factories? If so we should mention it here :)

Reply:
I´ve removed the `Factory` stuff from the docs altogether.